### PR TITLE
FIX Unknown alias: default (Psych::BadAlias)

### PR DESCRIPTION
**Pull request by ps-ruby**
Ruby 3.0 comes with Psych 3, while Ruby 3.1 comes with Psych 4, which has a major breaking change.
1. The new YAML loading methods (Psych 4) do not load aliases unless they get the aliases: true argument.
2. The old YAML loading methods (Psych 3) do not support the aliases keyword.

This change is breaking the codebase while loading the database.yml and this PR handles that for ruby 3.1.x

### CODE REVIEW

File: lib/kuby/plugins/rails_app/database.rb [hunk 1]
```diff
@@ -51,7 +51,11 @@ def db_config
         end
 
         def db_configs
-          @db_configs ||= YAML.load(File.read(db_config_path))
+          @db_configs ||= begin
```
Comment (camertron) [inline lib/kuby/plugins/rails_app/database.rb:54]: Can we check `Psych::VERSION >= '4'` here instead?

**Review by camertron — CHANGES_REQUESTED** [review]
Thanks for the PR @ps-ruby :)

File: lib/kuby/plugins/rails_app/database.rb (hunk 1 above)
Comment (ps-ruby) [reply lib/kuby/plugins/rails_app/database.rb:54]: Sure
I will make the change.

Comment (ps-ruby) [reply lib/kuby/plugins/rails_app/database.rb:55]: @camertron I have made the changes

**Comment by ps-ruby** [comment]
@camertron Sorbet check is failing for _lib/kuby/docker/remote_tags.rb_. I tried auto-correct and it is making the changes shown below. 

image

Should I make push the above changes?

**Comment by camertron** [comment]
@ps-ruby ah sorry about that. For some reason Sorbet isn't picking up that new class I added to `Docker::Remote`. I fixed it manually in master, so if you merge/rebase the tests should pass :)

**Review by camertron — APPROVED** [review]

**Pull request merged**